src/objects/deck/deck.scss

```diff
@@ -0,0 +1,52 @@
+@use '../../design-tokens/breakpoint.yml';
+@use '../../design-tokens/sizes.yml';
+@use '../../mixins/fluid';
+@use '../../mixins/media-query';
+
+/**
+ * 1. If horizontal items are shown at the wrong column count, they will appear
+ *    to break the grid. This rule keeps items densely packed so it will always
+ *    appear visually correct.
+ */
+
+.o-deck {
+  display: grid;
+  grid-auto-flow: dense; /* 1 */
```

Comment:
Do we have any concerns about this leading to mismatched ordering of articles for screen reader users?

Reply:
I re-read [CSS Grid Layout and Accessibility](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/CSS_Grid_Layout_and_Accessibility) and I think this qualifies as _visual_ rather than _logical_ re-ordering. Navigation through the document will continue to be chronological, and at most tabbing will be a single row off.

At one point I had this moved to a modifier class, but then I remembered this is not a general Grid component and I couldn't think of a use case where we _wouldn't_ want this. I'm open to further discussion, though!

Reply:
Cool, yeah it felt slightly jarring for me to jump down a row and then back up, but I don't think it's a major concern. I'm fine moving forward with this, but wanted to ensure we were thinking it through :+1: